packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/ParameterProvider.cs

```diff
@@ -63,33 +65,27 @@ public ParameterProvider(InputParameter inputParameter)
             Attributes = attributes ?? Array.Empty<AttributeStatement>();
         }
 
-        private ParameterValidationType GetParameterValidation(InputModelProperty property, CSharpType propertyType)
```

Comment:
why was parameter validation moved from the parameter provider onto the property provider?